packages/components/organisms/f-offers/src/components/Results.vue

```diff
@@ -0,0 +1,277 @@
+<template>
+    <content-cards
+        v-if="canShowContentCards"
+        #default="{ cards }"
+        :user-id="globalUserId"
+        :api-key="brazeApiKey"
+        :locale="$i18n.locale"
+        :push-to-data-layer="pushToDataLayer"
+        @has-loaded="hasLoaded"
+        @get-card-count="cardsReceived"
+        @on-error="onError"
+        @voucherCodeClick="openModal"
+    >
+        <div :class="$style['c-offersResults-contentCards']">
+            <template v-for="(card, i) in cards">
+                <group-header-card
+                    v-if="card.type === 'Header_Card'"
+                    :key="i"
+                    :title="card.title" />
+                <component
+                    :is="handleCustomCardType(card.type)"
+                    v-else
+                    :key="i"
+                    :card="card"
+                    :test-id="testIdForItemWithIndex(i)"
+                    :tenant="tenant"
+                />
+            </template>
+        </div>
+    </content-cards>
+</template>
+
+<script>
+import {
+    ContentCards,
+    PromotionCard,
+    VoucherCard,
+    GroupHeaderCard,
+    FirstTimeCustomerCard,
+    TermsAndConditionsCard,
+    SkeletonLoader
+} from '@justeat/f-content-cards';
+import '@justeat/f-content-cards/dist/f-content-cards.css';
+import { mapState } from 'vuex';
+import { VUEX_MODULE_NAMESPACE_OFFERS } from '../store/types';
+
+const MAX_TRIES = 3;
+const MAX_RELOAD_TIMEOUT = 10000; // 10 seconds
+
+export default {
+    name: 'OffersResults',
+
+    components: {
+        ContentCards,
+        PromotionCard,
+        VoucherCard,
+        GroupHeaderCard,
+        FirstTimeCustomerCard,
+        TermsAndConditionsCard,
+        SkeletonLoader
+    },
+
+    data: () => ({
+        enabledCardTypes: [
+            'Header_Card',
+            'Promotion_Card_1',
+            'Promotion_Card_2',
+            'Recommendation_Card_1',
+            'Restaurant_FTC_Offer_Card',
+            'Voucher_Card_1',
+            'Anniversary_Card_1'
+        ],
+        title: 'Offers',
+        hasMounted: false,
+        contentCardsHaveLoaded: false,
+        error: undefined,
+        showContentCards: true,
+        tries: 0,
+        isModalOpen: false,
+        modalOngoingUrl: '',
+        loadingCard: { type: 'postOrder', count: 3 },
+        pushToDataLayer: () => {}
+    }),
+
+    computed: {
+        ...mapState(VUEX_MODULE_NAMESPACE_OFFERS, [
+            'brazeApiKey',
+            'globalUserId'
+        ]),
+
+        /**
+         * Determines whether the component should attempt to show.
+         * @return {Boolean}
+         **/
+        canShowContentCards () {
+            return this.hasMounted // Only show if the component has mounted (i.e. not during SSR)
+                && this.globalUserId // Only attempt to show if we've retrieved a non-null userID
+                && this.showContentCards; // Allows card component to be toggled off in the event of load failure
```

Comment:
This prop feels to be redundant as you can change the value of canShowContentCards in the event of load failure but maybe I missed something

Reply:
I can agree to a point, but tbh your comment brings up some other interesting points, I think that results / f-offers should not be the one doing the check for braze loading correctly this should be handled inside of content cards itself I think. This way content cards can just provide a state variable via the scope slot that indicates the state, whether that's (error, loading, success)?

Reply:
On the above point this is already in content cards for stamp cards so I should just be following that, I will put do not merge on this for now.